tests/providers/google/cloud/operators/test_dataprep.py

```diff
@@ -0,0 +1,34 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from unittest import TestCase, mock
+
+from airflow.providers.google.cloud.operators.dataprep import DataprepGetJobsForJobGroupOperator
+
+JOB_ID = 143
+TASK_ID = "dataprep_job"
+
+
+class TestDataprepGetJobsForJobGroupOperator(TestCase):
+    @mock.patch(
+        "airflow.providers.google.cloud.operators." "dataprep.GoogleDataprepHook"
+    )
+    @mock.patch("airflow.providers.google.cloud.operators." "dataprep.BaseOperator")
```

Comment:
Why do we have to mock BaseOperator?

Reply:
yy... I was pretty sure that I have to mock base op. Ok, fixed.